### ISSUE TITLE
Crash on login linked to Journal Gump

### ISSUE THREAD

**Issue by dallenwilson** [ClassicUO/ClassicUO#1307]
**Describe the bug**
On logging in, after character selection, the game window flashes up for a moment (with everything in it's last saved position) and then the game crashes.

**To Reproduce**
Unable to reproduce on demand. I've gone a week without it happening, I've had it happen twice in two days. Has happened with several different ClassicUO versions now. I have copies of the offending file, ~~but I haven't tried restoring one to see if it triggers the crash again~~ but restoring an old copy does not trigger the crash. I re-named this morning's problem-file, logged in to the game and then logged out, closed the game and launcher, then replaced the newly-created gumps.xml with the previous one that had caused a crash. Started the launcher and logged in, no crash.

**Expected behavior**
At best I'd like it to not happen at all, but I'd settle for it just removing the journal gump and allowing play instead of a hard crash.

**Screenshots**
Game doesn't stay up long enough to take a screenshot. Here's the console output:
```
[ERROR] FATAL UNHANDLED EXCEPTION: System.ArgumentOutOfRangeException: Invalid existing index 8 for source length 8
Parameter name: index
  at ClassicUO.Utility.Collections.Deque`1[T].CheckExistingIndexArgument (System.Int32 sourceLength, System.Int32 index) [0x0002a] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.Utility.Collections.Deque`1[T].get_Item (System.Int32 index) [0x00006] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.Game.UI.Gumps.JournalGump+RenderedTextList.CalculateScrollBarMaxValue () [0x0001f] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.Game.UI.Gumps.JournalGump+RenderedTextList.Update (System.Double totalTime, System.Double frameTime) [0x00050] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.Game.UI.Controls.Control.Update (System.Double totalTime, System.Double frameTime) [0x00051] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.Game.UI.Gumps.Gump.Update (System.Double totalTime, System.Double frameTime) [0x00024] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.Game.UI.Gumps.JournalGump.Update (System.Double totalTime, System.Double frameTime) [0x00000] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.Game.Managers.UIManager.Update (System.Double totalTime, System.Double frameTime) [0x0001e] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.GameController.Update (Microsoft.Xna.Framework.GameTime gameTime) [0x000cf] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at Microsoft.Xna.Framework.Game.Tick () [0x001fe] in <140806325002443c8cfb12ff74fba306>:0 
  at Microsoft.Xna.Framework.Game.RunLoop () [0x0003c] in <140806325002443c8cfb12ff74fba306>:0 
  at Microsoft.Xna.Framework.Game.Run () [0x00032] in <140806325002443c8cfb12ff74fba306>:0 
  at ClassicUO.Client.Run () [0x00045] in <8eed2240175a42fa9efba27ca700a9a4>:0 
  at ClassicUO.Bootstrap.Main (System.String[] args) [0x002d3] in <8eed2240175a42fa9efba27ca700a9a4>:0
```

**Shard**
 - Name of the shard: Ultima Legacy Reborn. RunUO 2.2, Core.LBR.

**Information (please complete the following information):**
 - OS: Gentoo Linux 64-bit. Using latest mono version packaged for gentoo: 6.12.0.107.
 - ClassicUO Version: 0.1.7.64 targeting client.exe v4.0.11c
 - [x] Using DEV Version?
 - Launcher: classicuo_launcher

**Additional context**
 - How many clients do you run in parallel: Just the one.
 - Screen Resolution: Three-monitor setup, all different. Game window runs on the 1920x1080 main monitor.

**Comment by dallenwilson** [comment]
I may have narrowed this down a bit, I tend to leave the journal and skills gump with the little 'minimize' icon hanging off the top of my screen. I haven't seen this crash since I stopped doing that.

**Comment by dallenwilson** [comment]
I believe this to be fixed in commit f0d6547b3e43717b2686bf9a0b4c4e831f28fe70
